manifests/l/LukeBriggs/Pepys/1.5.3/LukeBriggs.Pepys.yaml

```diff
@@ -5,5 +5,5 @@ PackageIdentifier: LukeBriggs.Pepys
 PackageVersion: 1.5.3
 DefaultLocale: en-US
 ManifestType: version
-ManifestVersion: 1.0.0
+ManifestVersion: 1.0.1
```

Comment:
This key must have the value "1.0.0". The Microsoft community package repository validation pipelines also use this value to determine appropriate validation rules when evaluating this file.

Reply:
Also, 1.0.1 schema doesn't exist.

We're at 1.1.0 now.